src/main/resources/alma/index-config.json

```diff
@@ -1123,6 +1123,12 @@
             "q.all"
           ]
         },
+        "disambiguatingDescription": {
+          "type": "text",
+          "copy_to": [
+            "q.all"
```

Comment:
Are you certain to copy this to the Digibib query? See https://github.com/hbz/lobid-resources/pull/2177#pullrequestreview-2892919288 .

Reply:
Is this new field somewhow relevant for the union catalogue?